pkg/inventory/store/vm_inventory.go

```diff
@@ -49,20 +49,28 @@ func keyAndSpanSelectFuncVm(selectors *antreastorage.Selectors, key string, obj
 	if selectors.Key != "" && key != selectors.Key {
 		return false
 	}
+	if selectors.Label.Empty() && selectors.Field.Empty() {
+		// Match everything.
```

Comment:
This scenario corresponds `kubectl get vm -A -w`?

Reply:
Yes